### PR TITLE
[windows] Fix several tree tutorials

### PR DESCRIPTION
**Pull request by bellenot**
- Copy ${ROOTSYS}/test/$<CONFIG>/libEvent.dll to ${ROOTSYS}/test/
- Fix dll name in several tree tutorials

### CODE REVIEW

File: tutorials/tree/copytree.C [hunk 1]
```diff
@@ -11,7 +11,11 @@
 /// \author Rene Brun
 
 // Load the library at macro parsing time: we need this to use its content in the code
+#ifdef R__WIN32
+R__LOAD_LIBRARY($ROOTSYS/test/libEvent.dll)
```
[the remark this answers is not in the document]
Comment (bellenot) [reply tutorials/tree/copytree.C:15]: @pcanal it's only in the case of full PATH. I.e. `R__LOAD_LIBRARY(libTree)` should work

[the remark this answers is not in the document]
Comment (pcanal) [reply tutorials/tree/copytree.C:15]: Would it be hard to extend the fullpath case to support both reading `actual_full_pathname` and `input.$DllSuf`?  (and/or my question stands for the full path name case)?

[the remark this answers is not in the document]
Comment (bellenot) [reply tutorials/tree/copytree.C:15]: I'll have to investigate...

**Review by pcanal — APPROVED** [review]
LGTM.

File: tutorials/tree/copytree.C (hunk 1 above)
[the remark this answers is not in the document]
Comment (dpiparo) [reply tutorials/tree/copytree.C:15]: Does autoload of that lib fail?

[the remark this answers is not in the document]
Comment (pcanal) [reply tutorials/tree/copytree.C:15]: Autoload is not in play here as the library's directory is intentionally not on the library path.

**Pull request merged**